### ISSUE TITLE
Defining `_input` causes segfault

### ISSUE THREAD

**Issue by matheus2740** [touilleMan/godot-python#189]
```python
def _input(self, event):
    pass
```
causes:
```
[1] 1   libsystem_platform.dylib            0x00007fff51ba6f5a _sigtramp + 26
[2] 2   ???                                 0x00000000c2000000 0x0 + 3254779904
[3] Ref<InputEventMouseMotion>::Ref<InputEvent>(Ref<InputEvent> const&) (in Godot) + 49
[4] Viewport::_notification(int) (in Godot) + 5118
[5] Viewport::_notificationv(int, bool) (in Godot) + 76
[6] Object::notification(int, bool) (in Godot) + 19
[7] SceneTree::_notify_group_pause(StringName const&, int) (in Godot) + 2074
[8] SceneTree::iteration(float) (in Godot) + 538
[9] Main::iteration() (in Godot) + 531
[10] OS_OSX::run() (in Godot) + 391
[11] main (in Godot) + 529
[12] 12  libdyld.dylib                       0x00007fff51898015 start + 1
[13] 13  ???                                 0x0000000000000004 0x0 + 4
-- END OF BACKTRACE --
```

Additional info:
tool script (export(tool=True))
MacOS
Built from source (master) with full bindings

**Comment by matheus2740** [comment]
This is happening on windows too, in a normal script (not tool)

**Comment by matheus2740** [comment]
same thing on linux (Pop!_OS 20.04 x64 nvidia)

**Comment by matheus2740** [comment]
Looks like a bug in Godot. I just built godot from source (tags/3.2.1-stable) and there's no problem there.

**Comment by matheus2740** [comment]
Seems to be resolved with Godot 3.2.2 . if anyone else bumps into this please reopen this issue.
